docs/eventing/sinks/README.md

```diff
@@ -132,6 +132,7 @@ The `svc` in `http://event-display.svc.cluster.local` determines that the sink i
 
 | Name                                                                             | Maintainer | Description                          |
 |----------------------------------------------------------------------------------| -- |--------------------------------------|
+| [IntegrationSink](./integration-sink/README.md)                                  | Knative  | Send events to generic event sink    |
```

Comment:
```suggestion
| [IntegrationSink](./integration-sink/README.md)                                  | Knative  | Send events to a generic event sink    |
```

Reply:
Also, do we want to link to the specific IntegrationSinks here as well?

Reply:
not sure. yet